### PR TITLE
[WEB-4136]: add UpdateCartBillingAddress command + tests

### PR DESCRIPTION
**Pull request by aosmakserato**

### CODE REVIEW

File: src/Ecom/Command/UpdateCartBillingAddress.php [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    {
+        return 'PUT';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getUriPath(): string
+    {
+        return "/api/v1//carts/{$this->commandArgs['cart_uuid']}/billingaddress";
```
Comment (mshahidi) [inline src/Ecom/Command/UpdateCartBillingAddress.php:37]: Extra `/` after `v1`

File: tests/Ecom/Command/UpdateCartBillingAddressTest.php [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            'http://my.server.com',
+            $args
+        );
+
+        $request = $command->getRequest();
+        parse_str((string)$request->getBody(), $bodyParams);
+
+        $this->assertEquals('PUT', $request->getMethod());
+        $this->assertRegExp('/^Basic [[:alnum:]=]+$/', $request->getHeaderLine('Authorization'));
+        $this->assertStringEndsWith("/api/v1//carts/{$cartUuid}/billingaddress", $request->getUri()->getPath());
```
Comment (mshahidi) [inline tests/Ecom/Command/UpdateCartBillingAddressTest.php:72]: Test is also asserting the wrong URL 😄

Comment (aosmakserato) [reply tests/Ecom/Command/UpdateCartBillingAddressTest.php]: fixed

File: src/Ecom/Command/UpdateCartBillingAddress.php (hunk 1 above)
Comment (aosmakserato) [reply src/Ecom/Command/UpdateCartBillingAddress.php]: fixed

**Review by mshahidi — APPROVED** [review]

**Pull request merged**